GenderPayGap.WebUI/Views/ManageOrganisations/AllOrganisationReports.cshtml

```diff
@@ -0,0 +1,329 @@
+                                            </a>
+                                        </div>
+                                    }
+                                    @{ await Html.RenderPartialAsync("_ReportStatusBadge",
+                                           new ReportStatusBadgeViewModel
+                                           {
+                                               Desktop = false,
+                                               DateText = report.GetReportingDeadline().ToString("d MMMM yyyy"),
+                                               ReportStatus = report.GetReportBadge(),
+                                               MobileClasses = "govuk-!-margin-top-5 govuk-!-margin-bottom-3"
+                                           }); 
+                                    }
+                                    <br/>
+                                    <a loadtest-id="create-report-@report.ReportingYear" class="govuk-link"
+                                       href="@Url.Action("ReportOverview", "ReportOverview", new {encryptedOrganisationId, reportingYear = report.ReportingYear, canTriggerLateSubmissionWarning = true})">
+                                        @report.GetReportLinkText()
+                                    </a>
+                                </text>,
+                        Classes = "govuk-!-padding-bottom-8"
+                    }
+                }
+            };
+            return newRow;
+        }
+        
+        var dateJoinedRow = new TableRowViewModel
+        {
+            Row = new List<TableCellViewModel>
+            {
+                new TableCellViewModel
+                {
+                    Html = @<text>
+                               @await Html.GovUkHint(
+                                   new HintViewModel
+                                   {
+                                       Text = "Employer joined Gender Pay Gap service on " + @Model.Organisation.Created.ToString("dd/MM/yyyy"),
+                                       Classes = "govuk-!-margin-top-4 govuk-!-text-align-centre"
+                                   })
+                            </text>,
+                    Colspan = 4
+                }
+            }
+        };
+        
+        var indexOfCompanyJoined = organisation.GetRecentReports(Global.ShowReportYearCount).ToList().FindIndex(report => Model.Organisation.Created > ReportingYearsHelper.GetDeadlineForAccountingDate(report.AccountingDate));
+        if(indexOfCompanyJoined >= 0)
+        {
+            allMobileRows.Insert(indexOfCompanyJoined, dateJoinedRow);
+            allDesktopRows.Insert(indexOfCompanyJoined, dateJoinedRow);
+        }
+        else
+        {
+            allMobileRows.Add(dateJoinedRow);
+            allDesktopRows.Add(dateJoinedRow);
+        }
+        
+        var currentPage = Model.CurrentPage ?? 1;
+        var entriesPerPage = Model.EntriesPerPage ?? 10;

```

Comment:
Pretty sure these are both set correctly by the controller, do we need the null checking here?

Reply:
This was purely because I was in the habit of setting for loops with an `int i` but that can be `var` however I believer I'll be removing this for loop as per another comment anyway - 
![image](https://user-images.githubusercontent.com/62190777/201137528-7c0b7c2d-4b41-4a6d-898d-052c1ba6c271.png)